js/PulsarUIComponent.js

```diff
@@ -158,22 +159,62 @@ PulsarUIComponent.prototype.initDataTables = function () {
         $($.fn.dataTable.tables(true)).DataTable().columns.adjust().responsive.recalc();
     });
 
+    // Dropdowns within responsive tables will be hidden by overflows, this method detaches the dropdown menu and reattaches it on the body
+    $(document).on('show.bs.dropdown', '.datatable', function(e) {
+        var dropdown = $(e.target),
+            dropdownMenu = dropdown.find('.dropdown__menu'),
+            dropdownMenuTop = dropdown.offset().top,
+            dropdownMenuLeft = dropdown.offset().left;
```

Comment:
Really tiny, but I'm fairly certain `getBoundingClientRect` (which is what jQuery will be using here) triggers layout/reflow, so we should aim to limit it's use as much as possible. Could we instead store the offset in memory and access the properties from that? Something like...


```
dropdownMenuOffset = dropdown.offset(),
dropdownMenuOffsetTop = dropdownMenuOffset.top,
dropdownMenuOffsetLeft = dropdownMenuOffset.top;
```